redos/7.3/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM centos:7 as prepare
+MAINTAINER Sergei Vorontsov "piligrim@rootnix.net"
+
+ENV REPO_URL=http://mirror.yandex.ru/redos/7.3/x86_64/os/ \
+    CHROOT_DIR="/chroot"
+
+RUN mkdir -p $CHROOT_DIR
+
+#Download any available meta package redos-release
+RUN for rpm in `curl -s $REPO_URL | grep 'redos-release' | sed 's/<.*>\(.*\)<\/.*>/\1/' | awk '{print $1}'`; \
+    do \
+        if [[ $rpm =~ redos-release-[0-9].[0-9]-([0-9]|[0-9][0-9]).el7.noarch.rpm ]]; \
+        then \
+            curl -o $CHROOT_DIR/redos-release.rpm $REPO_URL$rpm; \
+        fi; \
+    done
+
+#setup base system
```

Comment:
Please keep consistency around the comments: `s` -> `S`, add a space.

Reply:
I have splitted this block and expended comments.